common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -141,6 +146,10 @@ def setUp(self):
         self.rate_limit, rate_duration = throttle.parse_rate(throttle.rate)
 
         self.course = CourseFactory.create()
+        # Load a CourseOverview. This initial load should result in a cache
+        # miss; the modulestore is queried and course metadata is cached.
+        __ = CourseOverview.get_from_id(self.course.id)
```

Comment:
@clintonb this inserts a row into the CourseOverview table for the newly-created course, which requires a modulestore query. With this cache now warm, later calls to CourseOverview should not make any modulestore queries, behavior which the tests verify.